content/contact.md

```diff
@@ -5,8 +5,10 @@ omit_header_text: true
 description: We'd love to hear from you
 type: page
 menu: main
----
+weight: 50
 
-This is an example of a custom shortcode that you can put right into your content. You will need to add a form action to the shortcode to make it work. Check out [Formspree](https://formspree.io/) for a simple, free form service. 
+---
+<!-- 
+This is an example of a custom shortcode that you can put right into your content. You will need to add a form action to the shortcode to make it work. Check out [Formspree](https://formspree.io/) for a simple, free form service.  -->
 
-{{< form-contact action="https://example.com"  >}}
+{{< form-contact action="t4eq.community@gmail.com"  >}}
```

Comment:
I don't think this works as you intended. Currently it redirects to `<url>/contact/t4eq.community@gmail.com`

As I understand, the suggestion to use `Formspree` is to avoid having to write custom `js` code to handle the form data submission by using a free 3rd party service.

Reply:
AFAIU we need to register with formspree and then link here whatever link we get from formspree after registering/configuring the dialog. But let's maybe handle this separately in another PR. I can have a look at it.